packages/dev/docs/pages/react-spectrum/ssr.mdx

```diff
@@ -293,64 +293,24 @@ Media queries and DOM feature detection cannot be performed on the server becaus
 
 [Next.js](https://nextjs.org) is a framework for building websites and web applications with React. It supports both server side rendering as well as static rendering. A small amount of configuration is required to get React Spectrum’s CSS working with Next.js.
 
-Add the following to your `next.config.js` file. This will ensure that React Spectrum’s CSS is loaded properly by Next.js. Note that packages may need to be removed or added to the config below if using an older or newer version of React Spectrum.
+Add the following to your `next.config.js` file. This will ensure that React Spectrum’s CSS is loaded properly by Next.js. Note that packages may need to be added to the config below depending on your version of React Spectrum.
 
 ```tsx
 module.exports = {
   transpilePackages: [
     "@adobe/react-spectrum",
-    "@react-spectrum/actionbar",
-    "@react-spectrum/actiongroup",
-    "@react-spectrum/avatar",
-    "@react-spectrum/badge",
-    "@react-spectrum/breadcrumbs",
-    "@react-spectrum/button",
-    "@react-spectrum/buttongroup",
-    "@react-spectrum/calendar",
-    "@react-spectrum/checkbox",
-    "@react-spectrum/combobox",
-    "@react-spectrum/contextualhelp",
-    "@react-spectrum/datepicker",
-    "@react-spectrum/dialog",
-    "@react-spectrum/divider",
-    "@react-spectrum/dnd",
-    "@react-spectrum/form",
-    "@react-spectrum/icon",
-    "@react-spectrum/illustratedmessage",
-    "@react-spectrum/inlinealert",
-    "@react-spectrum/image",
-    "@react-spectrum/label",
-    "@react-spectrum/labeledvalue",
-    "@react-spectrum/layout",
-    "@react-spectrum/link",
-    "@react-spectrum/list",
-    "@react-spectrum/listbox",
-    "@react-spectrum/menu",
-    "@react-spectrum/meter",
-    "@react-spectrum/numberfield",
-    "@react-spectrum/overlays",
-    "@react-spectrum/picker",
-    "@react-spectrum/progress",
-    "@react-spectrum/provider",
-    "@react-spectrum/radio",
-    "@react-spectrum/slider",
-    "@react-spectrum/searchfield",
-    "@react-spectrum/statuslight",
-    "@react-spectrum/switch",
-    "@react-spectrum/table",
-    "@react-spectrum/tabs",
-    "@react-spectrum/tag",
-    "@react-spectrum/text",
-    "@react-spectrum/textfield",
-    "@react-spectrum/theme-dark",
-    "@react-spectrum/theme-default",
-    "@react-spectrum/theme-light",
-    "@react-spectrum/tooltip",
-    "@react-spectrum/view",
-    "@react-spectrum/well",
-    "@spectrum-icons/illustrations",
-    "@spectrum-icons/ui",
-    "@spectrum-icons/workflow"
+
+    // add required @react-spectrum/<name> packages, such as:
```

Comment:
I like how you added this as an option via a comment. There is another related PR ([replace package enumerations with glob-stars](https://github.com/adobe/react-spectrum/pull/6176/files#top)) that is using glob to manage these packages. Could we update your PR to use glob and `@react-spectrum/*`? It would automatically include all the packages without people needing to update their config.

Reply:
Done. (Thanks for mentioning that)

Reply:
Thank you!